package.json

```diff
@@ -24,7 +24,8 @@
     "tarn": "^1.1.5",
     "tildify": "1.2.0",
     "uuid": "^3.3.2",
-    "v8flags": "^3.1.3"
+    "v8flags": "^3.1.3",
+    "webpack": "^4.30.0"
```

Comment:
wait, webpack is a runtime dependency?..

Reply:
it appeared there master merge / rebase mess. I thought it was some, maybe typescript related stuff that other people had contributed lately... but anyways it shouldn't be in this PR.